logging/monolog_console.rst

```diff
@@ -35,16 +35,22 @@ current log level and the console verbosity.
 
 The example above could then be rewritten as::
 
+    use Psr\Log\LoggerInterface;
     use Symfony\Component\Console\Input\InputInterface;
     use Symfony\Component\Console\Output\OutputInterface;
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    private $logger;
```

Comment:
This is becoming quite unreadable. Can we please at the class definition (`class .... { }`) lines?